.openshiftio/application.yaml

```diff
@@ -21,7 +21,7 @@ parameters:
     value: .
     required: true
   - name: RELEASE_VERSION
-    description: The location within the source repo of the application
+    description: The version used to tag image artifacts
     displayName: release version
     value: '1'
```

Comment:
can this be the value of the version in the package.json.  i think it is 2.0.1

Reply:
Sure https://github.com/nodeshift-starters/nodejs-rest-http/pull/120/files#diff-5dce3a731b5aff7080a31a1adcd60ae7